gwnr/waveform/esigma_utils.py

```diff
@@ -302,7 +302,7 @@ def get_inspiral_esigma_modes(
             k: pt.TimeSeries(
                 modes[k].data.data,
                 delta_t=delta_t,
-                epoch=-delta_t * len(modes[k].data.data),
+                epoch=-delta_t * (len(modes[k].data.data)-1),
```

Comment:
Why is this `-1` needed?

Reply:
Consider a time-grid of $N$ points with grid-spacing $\Delta t$. If the last grid point is to be at $0$ (i.e. $t_{N-1}=0$), then the formula for time at the $i^{\rm{th}}$ grid point ($t_i$) would be,

$t_i = -\Delta t \ (N-1-i); i \in \lbrace 0, 1, 2, ..., N-1 \rbrace$

which gives $t_0 = -\Delta t*(N-1)$.